x-pack/solutions/search/plugins/search_indices/public/code_examples/javascript.ts

```diff
@@ -149,3 +149,41 @@ const updateMappingResponse = await client.indices.putMapping({
 });
 console.log(updateMappingResponse);`,
 };
+
+export const JSSemanticIngestDataExample: IngestDataCodeDefinition = {
+  ...JSIngestDataExample,
+  ingestCommand: ({
+    apiKey,
+    elasticsearchURL,
+    sampleDocuments,
+    indexName,
+  }) => `const { Client } = require('@elastic/elasticsearch');
+
+const client = new Client({
+  node: '${elasticsearchURL}',
+  auth: {
+    apiKey: '${apiKey ?? API_KEY_PLACEHOLDER}'
+  },
+});
+
+// Set the ingestion timeout to 5 minutes, to allow for semantic ingestion
```

Comment:
@leemthompo copy check here, I'm thinking about how I can i18n this, but that would mean it would have to be single line vs this cleaner multi-line comment.

Reply:
I like that I would probably swap ingestion and allocation:

`Timeout to allow ML node allocation and semantic ingestion to complete`

Since thats the order in which they happen.

Reply:
Perfect! I had the same thought an hour later after I had finished my day 😁